### ISSUE TITLE
hub.docker.com cookies consent dialog delays site access unless full consent is given

### ISSUE THREAD

**Issue by RompitaKablo** [docker/hub-feedback#2169]
It seems that when first accessing Docker Hub and selecting "Essential Cookies Only" a dialog box appears and causes an unnecessarily long delay, using a "progress timer", before the site can be accessed. Is this is method a timed "cookie wall", employed to discourage users from allowing anything but all cookies, or is there valid a technical reason that a user choice limiting cookies cannot be recorded as quickly as the "Accept all cookies" option is?

[ICO guidance states](https://ico.org.uk/for-organisations/guide-to-pecr/guidance-on-the-use-of-cookies-and-similar-technologies/how-do-we-comply-with-the-cookie-rules/#comply7), "Recital 32 of the UK GDPR is clear that electronic consent requests must not be unnecessarily disruptive – so you need to consider how you go about providing clear and comprehensive information without confusing users or disrupting their experience."

It seems that this matter has been raised before but ignored.

https://github.com/docker/hub-feedback/issues/2061

**Comment by janissimsons** [comment]
Yes, I also noticed this behaviour. They are trying to force users to consent to all cookies, which is illegal in the European Union.
Accepting and rejecting cookies must take the same amount of time. It's not a recommendation but a very clear law.
